index.js

```diff
@@ -7,33 +7,8 @@ module.exports = {
   included: function(app) {
     this._super.included(app);
 
-    // lodash
-    app.import({
-      development: 'node_modules/lodash/lodash.js',
-      production:  'node_modules/lodash/lodash.min.js'
-    });
-
-    // jquery-ui dependencies needed by gridstack.js
-    // https://github.com/gridstack/gridstack.js/blob/v0.2.6/src/gridstack.js#L10
-    [ 'version', 'data', 'disable-selection', 'focusable', 'escape-selector', 'form',
-      'ie', 'keycode', 'labels', 'jquery-1-7', 'plugin', 'safe-active-element',
-      'safe-blur', 'scroll-parent', 'tabbable', 'unique-id', 'widget'
-    ].forEach(function(module) {
-      app.import({
-        development: 'node_modules/jquery-ui/ui/' + module + '.js',
-        production:  'node_modules/jquery-ui/ui/' + module + '.js'
-      });
-    });
-
-    [ 'mouse', 'draggable', 'droppable', 'resizable' ].forEach(function(module) {
-      app.import({
-        development: 'node_modules/jquery-ui/ui/widgets/' + module + '.js',
-        production: 'node_modules/jquery-ui/ui/widgets/' + module + '.js'
-      });
-    });
-
     // Gridstack
-    [ 'gridstack', 'gridstack.jQueryUI' ].forEach(function(module) {
+    [ 'gridstack', 'gridstack.jQueryUI', 'jquery-ui' ].forEach(function(module) {
```

Comment:
you may want to import in order (and one missing)
gridstack-poly.js (tiny file we need for IE11)', 'gridstack.js', 'jquery-ui.js', 'gridstack.jQueryUI.js

there is ALSO gridstack-extra.css (line #17 below) IF you need to support [2-11] columns (and min version of all files). Might want to update that as well.